include/allegro5/monitor.h

```diff
@@ -16,6 +16,7 @@ typedef struct ALLEGRO_MONITOR_INFO
    int y1;
    int x2;
    int y2;
+   int dpi;
```

Comment:
maybe this should be float?

Reply:
Although I think it makes more sense to represent the number of physical dots as an integer, I just checked xrandr source code for reference, and it uses `double` there. So, I could as well change this.